validator/testcases/regex.py

```diff
@@ -2901,9 +2915,34 @@ def munge_filename(name):
                         "the `%s` preference branch" % branch)))
         for branch, reason in BANNED_PREF_BRANCHES))
 
-STRING_REGEXPS += PREF_REGEXPS
+# For tests in literal strings, add help text suggesting passing the
+# preference directly to reference getter functions.
+PREF_STRING_HELP = (
+    "If you are reading, but not writing, this preference, please consider "
+    "passing a string literal directly to `Preferences.get()` or "
+    "`nsIPrefBranch.get*Pref`.")
+
+
+def maybe_tuple(value):
+    if isinstance(value, tuple):
+        return value
+    return (value,)
```

Comment:
The `if` isn't some special early return, it is one of the two cases. If we handles a third case for `list` then we'd get:

```py
def to_tuple(value):
    if isinstance(value, tuple):
        return value
    elif isinstance(value, list):
        return tuple(*value)
    return (value,)
```

I don't know what makes that last case special, it's just the default so it goes in the `else`.